substrate-processor/src/interfaces/handlerContext.ts

```diff
@@ -28,6 +28,14 @@ export interface EventHandler {
 }
 
 
+export interface EvmLogHandlerContext extends EventHandlerContext {
```

Comment:
Should be expanded. The suggestion is to keep the existing `SubstrateBlock`, `SubstrateEvent` interfaces intact.
But, evm specific handlers we'll get additional data.